README.md

```diff
@@ -0,0 +1,308 @@
+# Greenfield BSC Archiver
+
+## Description
+
+The Greenfield BSC Archiver is tasked with storing headers and bodies of BSC historical blocks. It uploads this block information to Greenfield via the bundle service. The components of this archiver include:
+
+### Syncer
+
+Responsible for fetching block information via Ethereum calls, storing it in a database, and uploading it to Greenfield through the bundle service. The current setting uploads data every 100 blocks.
+
+### Verifier
+
+Regularly checks the integrity of uploaded data on Greenfield. If errors are found, it corrects and re-uploads the data, and updates the block bundle name in the database.
+
+### Server
+
+Provides RPC interfaces compatible with Ethereum calls. Variables are generated using a Swagger.yaml file to ensure adaptability.
+1. eth_getBlockByNumber
+2. eth_getBundledBlockByNumber
+3. eth_blockNumber
+4. eth_getBlockByHash
+5. /bsc/v1/blocks/{block_id}/bundle/name
+
+### Example
+
+**eth_getBlockByNumber**
+request
+```json
+{
+    "method": "eth_getBlockByNumber",
+    "params": [
+        "0x2625A00"
+    ],
+    "id": 100,
+    "jsonrpc": "2.0"
+}
+```
+
+
+response
+```json
+{
+    "id": 100,
+    "jsonrpc": "2.0",
+    "result": {
+        "difficulty": "0x2",
+        "extraData": "0xd883010002846765746888676f312e31332e34856c696e757800000000000000924cd67a1565fdd24dd59327a298f1d702d6b7a721440c063713cecb7229f4e162ae38be78f6f71aa5badeaaef35cea25061ee2100622a4a1631a07e862b517401",
+        "gasLimit": "0x25ff7a7",
+        "gasUsed": "0x300b37",
+        "hash": "0x04055304e432294a65ff31069c4d3092ff8b58f009cdb50eba5351e0332ad0f6",
+        "logsBloom": "0x08000000000000000000000000000000000000000000000000000000000000000000000000000000000000000020000000000000100000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000300000000000000000000000000000",
+        "miner": "0x2a7cdd959bFe8D9487B2a43B33565295a698F7e2",
+        "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+        "nonce": "0x0000000000000000",
+        "number": "0x1",
+        "parentHash": "0x0d21840abff46b96c84b2ac9e10e4f5cdaeb5693cb665db62a2f3b02d2d57b5b",
+        "receiptsRoot": "0xfc7c0fda97e67ed8ae06e7a160218b3df995560dfcb209a3b0dddde969ec6b00",
+        "sha3Uncles": "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347",
+        "size": "0x558",
+        "stateRoot": "0x1db428ea79cb2e8cc233ae7f4db7c3567adfcb699af668a9f583fdae98e95588",
+        "timestamp": "0x5f49ca59",
+        "totalDifficulty": "0x3",
+        "transactions": [
```

Comment:
The example is too long, we dont have to show all of them. It can improve the readability of the readme.

Reply:
fixed